products/podman-engine.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Podman Engine
+category: app
+iconSlug: podman
+permalink: /podman-engine
+alternate_urls:
+-   /podman
+versionCommand: podman --remote version --format '{{.Server.Version}}'
+releaseDateColumn: true
+
+identifiers:
+-   repology: podman
+
+auto:
+  methods:
+  -   git: https://github.com/containers/podman.git
+      regex: ^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$
```

Comment:
```suggestion
```

The default regex should work with podman's tags.

Reply:
Makes sense, I think I was messsing around with the (not needed) `-rhel` tags.